### PR TITLE
qt: Hide remaining PrivateSend UI if PrivateSend is not enabled 

### PR DESCRIPTION
**Pull request by xdustinface**

### CODE REVIEW

**Comment by UdjinM6** [comment]
Pls see 613552d9cd

File: src/qt/transactionview.cpp [hunk 1]
```diff
@@ -742,3 +746,16 @@ void TransactionView::updateWatchOnlyColumn(bool fHaveWatchOnly)
     watchOnlyWidget->setVisible(fHaveWatchOnly);
     transactionView->setColumnHidden(TransactionTableModel::Watchonly, !fHaveWatchOnly);
 }
+
+void TransactionView::updatePrivateSendVisibility()
+{
+    bool fEnabled = CPrivateSendClientOptions::IsEnabled();
+    // If PrivateSend gets enabled use "Most common" else "All"
+    typeWidget->setCurrentIndex(fEnabled ? 1 : 0);
```
Comment (UdjinM6) [inline src/qt/transactionview.cpp:754]: It feels like this should be the other way around tbh
```suggestion
    // If PrivateSend gets enabled use "All" else "Most common"
    typeWidget->setCurrentIndex(fEnabled ? 0 : 1);
```

Comment (xdustinface) [reply src/qt/transactionview.cpp:754]: oh right, sounds better 😄

**Review by UdjinM6 — APPROVED** [review]
ACK

**Review by PastaPastaPasta — APPROVED** [review]
ACK, code review, and works as expected

**Pull request merged**